apps/native-component-list/src/screens/Clipboard/ClipboardScreen.tsx

```diff
@@ -0,0 +1,212 @@
+      ],
+    },
+    {
+      name: 'options',
+      type: 'object',
+      properties: [
+        {
+          name: 'inputFormat',
+          type: 'enum',
+          platforms: ['ios', 'android'],
+          values: [
+            { name: 'StringFormat.PLAIN_TEXT', value: Clipboard.StringFormat.PLAIN_TEXT },
+            { name: 'StringFormat.HTML', value: Clipboard.StringFormat.HTML },
+          ],
+        },
+      ],
+    },
+  ],
+  actions: (value: string, options: { inputFormat: Clipboard.StringFormat }) =>
+    Clipboard.setStringAsync(value, options),
+};
+
+const GET_STRING_ASYNC_CONFIG: FunctionDescription = {
+  name: 'getStringAsync',
+  parameters: [
+    {
+      name: 'options',
+      type: 'object',
+      properties: [
+        {
+          name: 'preferredFormat',
+          type: 'enum',
+          platforms: ['ios', 'android'],
+          values: [
+            { name: 'StringFormat.PLAIN_TEXT', value: Clipboard.StringFormat.PLAIN_TEXT },
+            { name: 'StringFormat.HTML', value: Clipboard.StringFormat.HTML },
+          ],
+        },
+      ],
+    },
+  ],
+  actions: async (options: { preferredFormat: Clipboard.StringFormat }) => {
+    const result = await Clipboard.getStringAsync(options);
+    return result.length > 0 ? result : '[nothing]';
+  },
+};
+
+const SET_IMAGE_ASYNC_CONFIG: FunctionDescription = {
+  name: 'setImageAsync',
+  platforms: ['ios', 'android'],
+  parameters: [
+    {
+      name: 'imageBase64Data',
+      type: 'constant',
+      value: '[selected from image picker]',
+    },
+  ],
+  actions: async () => {
+    const { granted } = await ImagePicker.requestMediaLibraryPermissionsAsync();
+    if (granted) {
+      const result = await ImagePicker.launchImageLibraryAsync({
+        mediaTypes: ImagePicker.MediaTypeOptions.Images,
+        base64: true,
+      });
+      if (!result.cancelled && result.base64) {
+        await Clipboard.setImageAsync(result.base64);
+        return 'Image copied to clipboard';
+      }
+    } else {
+      Alert.alert('Permission required!', 'You must allow accessing images in order to proceed.');
+    }
+    return 'Image not selected';
+  },
+};
+
+const GET_IMAGE_ASYNC_CONFIG: FunctionDescription = {
+  name: 'getImageAsync',
+  platforms: ['ios', 'android'],
+  parameters: [
+    {
+      name: 'options',
+      type: 'object',
+      properties: [
+        {
+          name: 'format',
+          type: 'enum',
+          values: [
+            { name: "'png'", value: 'png' },
+            { name: "'jpeg'", value: 'jpeg' },
+          ],
+        },
+        {
+          name: 'jpegQuality',
+          type: 'enum',
+          values: [
+            { name: '1', value: 1 },
+            { name: '0.5', value: 0.5 },
+            { name: '0', value: 0 },
+          ],
+        },
+      ],
+    },
+  ],
+  actions: Clipboard.getImageAsync,
+};
+
+const SET_URL_ASYNC_CONFIG: FunctionDescription = {
+  name: 'setUrlAsync',
+  platforms: ['android'],
```

Comment:
```suggestion
  platforms: ['ios'],
```
leftover from experiments